README.md

```diff
@@ -60,7 +60,7 @@ Once the installation is done, the next thing that you are going to do is to con
 
 #### Enable payment methods
 
-> Note that Alipay and Internet Banking payment methods are only available for merchants with a Thai-registered Omise account.
+> Note that Alipay, Bill Payment: Tesco, Installment, and Internet Banking payment methods are only available for merchants with a Thai-registered Omise account.
 
 There are 3 payment methods that you can enable on your store, which are **Alipay**, **Credit / Debit Card**, and **Internet Banking**.
```

Comment:
Do we need to add those payment methods here too?

Reply:
👍 let me update screenshot for this section too.